WalletWasabi/Microservices/ProcessAsync.cs

```diff
@@ -90,8 +90,7 @@ public virtual async Task WaitForExitAsync(CancellationToken cancellationToken,
 
 			try
 			{
-				Logger.LogTrace($"Wait for the process to exit: '{Process.StartInfo.FileName}'");
-
+				Logger.LogTrace($"Wait for the process to exit: '{Process.Id}'");
```

Comment:
note: This is required because `Process.StartInfo` cannot be accessed for a process accessed by its PID.